### DISCUSSION TITLE
required Matlab Version and more in-depth tutorials

### DISCUSSION THREAD

**Discussion by eSchulz7** [Open-Systems-Pharmacology/Forum#861] [Questions & Problems]
Dear Community,

I'm a master student and currently working on a project to build a model and simulate the tyrosin distribution in different organs. 

Therefore I would like to use the OSP Suite, but found the tutorials provided to be rather introductory and basic. Are there more complex explanatory examples, tutorials or maybe workshops, to get into the more complex model building, especially with MoBi and the matlab toolbox?

And to be able to use the matlab toolbox, which version of matlab is required? Is a student-version enough?

Many thanks in advance!

Elena Schulz

**Comment by StephanSchaller** [comment]
Dear Elena,

Matlab is no longer supported (it still works, but no support / further development of the toolbox is provided).
We recommend to use R instead.

Please check this tutorial for an introduction to MoBi:
https://github.com/Open-Systems-Pharmacology/PAGE2017-Budapest/tree/master/Exercises/5.%20Advanced%20PBPKPD%20modeling

Best,
Stephan
